### ISSUE TITLE
Consider turning "Change duration when changing start time" off by default

### ISSUE THREAD

**Issue by skel35** [toggl-open-source/toggldesktop#4015]
### 💻 Environment
Platform: Windows

### 📒 Description
"Change duration when changing start time" is on by default and it often causes confusion for the users. Consider use cases, user's feedback, and analytics data to make a decision on which value should be the default one.

**Comment by IndrekV** [comment]
As we want to unify it would make sense to check the behavior in web as a big factor.

**Comment by skel35** [comment]
You're right. Now I think I've described the issue too narrow. Defining such behavior should be one of the outcomes of the time input unification project.
